src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,68 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeData(getData(fromFileName),toFileName);
+    }
```

Comment:
```suggestion
		String rawData = readFromFile(fromFileName);
		String report = formReport(rawData);
        writeData(report, toFileName);
    }
```
я б таку структуру зробив методів